fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -742,3 +761,36 @@ private[this] final class Trie[T](strings: Seq[(String, T)]){
   }
 }
 
+
+
+trait TrueColor extends Category {
+  /**
+    * Selected Graphic Rendition
+    * 38 : front color
+    * 48 : background color
+    */
+  val trueColorCode : Int
+
+  def True(hex: Long) = {
+    val r = hex >> 16
+    val g = (hex & 0x00FF00) >> 8
+    val b = hex & 0x0000FF
+    makeAttr("\u001b[" + trueColorCode + ";2;" + r + ";" + g + ";" + b + "m", 274 + hex)("True(" + r + "," + g + "," + b +")")
```

Comment:
Rather than duplicating this logic between here and the overload below, can we extract it into a helper function `private[this] def True0(r: Int, g: Int, b: Int, index: Int)`?